### ISSUE TITLE
PathVisio is not compatible with Java 9

### ISSUE THREAD

**Issue by JonathanMELIUS** [PathVisio/pathvisio#58]
`Unresolved constraint in bundle `

**Comment by simon685** [comment]
I am having a similar problem. Only began recently. 

Starting from command line I get the following exception.
org.osgi.framework.BundleException: Unresolved constraint in bundle org.w3c.css.sac [26]: Unable to resolve 26.0: missing requirement [26.0] osgi.ee; (|(&(osgi.ee=JavaSE)(version=1.3.0))(&(osgi.ee=CDC/Foundation)(version=1.0.0)))

The java message is;
org.osgi.frameowrk.BundleException: Unresolved bundle in org.pathvisio.gui [10]

**Comment by mkutmon** [comment]
http://www.oracle.com/technetwork/java/javase/downloads/jdk9-downloads-3848520.html 
Java 9 not supported anymore - will test Java 10 now.

**Comment by DeniseSl22** [comment]
We should add Java 8 automatically when you install PV (just as Cytoscape does that).

**Comment by markwoon** [comment]
The current Java LTS is v11.  Are there any plans to support this?  

What exactly is tying Pathvisio to Java 8?  I rely on Pathvisio libraries directly and it is my only dependency that's blocking me from upgrading.

**Comment by mkutmon** [comment]
Yes, I hope to release the next version end of the month for Java 11

**Comment by markwoon** [comment]
Awesome!

Is this going to be PV4 or an update to PV3?

**Comment by mkutmon** [comment]
Bigger update to PV3 but still need to check some things related to dependencies of plugins. So it might have it's own plugin repo (not sure yet).

**Comment by markwoon** [comment]
Thanks.  Looking forward to it.

**Comment by mkutmon** [comment]
running into issues with some of the dependencies... I'll keep you posted but might take a little longer :(

**Comment by markwoon** [comment]
Just checking in to see how close this is to being released?

Curious as to what dependencies are causing problems...

**Comment by markwoon** [comment]
@mkutmon Any progress on this?  

We're at the point now where things we depend on are starting to require Java 11 minimum.

Are there any concrete plans to release an updated version of PathVisio in the near future?
